src/main/java/io/singularitynet/Client/VereyaModClient.java

```diff
@@ -182,11 +182,16 @@ public InputType getInputType() {
     // Control overriding:
     enum InputType
     {
-        HUMAN, AI, HYBRID;
-        private static final InputType[] vals = values();
+        HUMAN, AI, HYBRIDREG, HYBRIDMOUSE;
```

Comment:
What means REG?

Reply:
Also why MOUSE? Keyboard is also working in this mode.

Reply:
@noskill I'll hear your suggestions on names. REG means regular, MOUSE means that mouse control also enabled.

Reply:
To suggest a name i need to know what is idea behind these variables. What means "Regular"? Does it mean that mouse is not used in this mode?

Reply:
@noskill as I've said in the hangout chat and on the today's meeting, there are two options right now - hybrid mode as before (without mouse) and hybrid mode, that you've made on your PR to my branch. So yes, Regular means that with HybridReg mode on we can adjust AI-driven agent using keys only. With HybridMouse mode we can help AI-driven agent with mouse too.

Reply:
Good name for HYBRIDREG would HYBRID_KEYBOARD and just HYBRID instead of HYBRIDMOUSE, or bit more verbose HYBRID_MOUSE_KEYBOARD

Reply:
@noskill 
Ok, I've changed names to your suggestions. Can you please take a look at a problem with exiting to main menu, which I've mentioned on the meeting and in hangouts chat?